### ISSUE TITLE
Simply Getting The Password From Victim's Authentication?

### ISSUE THREAD

**Issue by Aphrodites1995** [wifiphisher/wifiphisher#1170]
I believe it might be possible to do an offline attack by asking the victim's password directly:
1. The victim sees the phishing wifi.
2. The victim connects to it because it has the same name as the real one.
3. The victim sends an encrypted password.
4. The phishing wifi receives the encrypted password, proceeds to bruteforce it by hashing a wordlist.
I did see this: https://github.com/wifiphisher/wifiphisher/issues/709 which is what I am talking about, but it is only suggesting receiving the password directly, and not bruteforcing it. Is there such an option?

**Comment by Bsoft2007** [comment]
You can just capture a handshake from original Wi-Fi and then brute force it offline with aircrack or Hashcat(faster using gpu)

**Comment by Amurmurmur** [comment]
@Bsoft2007 I think what @Aphrodites1995 tried to say, is that instead of bruteforcing is there a way to show a fake wifi login form to grab the password that user puts, it might be username and password in case of enterprise.
Sometimes bruteforcing is not trivial.

**Comment by sophron** [comment]
@Aphrodites1995 Yes, you can brute-force the password using a wordlist. This is common and can be done using various tools.

@Amurmurmur The password is never transmitted in the air: https://wifiphisher.readthedocs.io/en/latest/faq.html
